pandas/core/ops.py

```diff
@@ -990,6 +991,93 @@ def _construct_divmod_result(left, result, index, name, dtype):
     )
 
 
+def dispatch_to_extension_op(left, right, op_name=None, is_logical=False):
```

Comment:
the `dispatch_to_index_op` uses `op` instead of `op_name`. Is there a reason for this difference? (and I mean in the actual implementation here it assumes a method name, and not an operator function that can be called)

Reply:
If the op is not defined on ExtensionArray, calling it directly (`op(left_values, right_values)`) will raise a TypeError that you can catch (which you already do), so I don't really see the difference